connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/model/bop/SendEmailsRequest.java

```diff
@@ -11,9 +11,14 @@
 public class SendEmailsRequest {
```

Comment:
Even if this may never happen, let's make sure we'll never pass a `null` value in `emailSender` or `defaultRecipient`.
```suggestion
@JsonInclude(NON_NULL)
public class SendEmailsRequest {
```

Reply:
Done!